### PR TITLE
core: use `resolvedConfig` to name every resolved config

### PR DESCRIPTION
**Pull request by adamraine**
https://github.com/GoogleChrome/lighthouse/pull/14547#issuecomment-1334527580

Type renames will be done in a follow-up

### CODE REVIEW

**Comment by connorjclark** [comment]
Link to discussion?

**Comment by adamraine** [comment]
https://github.com/GoogleChrome/lighthouse/pull/14547#issuecomment-1334527580

File: core/test/config/config-test.js [hunk 1]
```diff
@@ -51,36 +51,36 @@ describe('Fraggle Rock Config', () => {
...
       gatherMode,
       configJson,
       {maxWaitForFcp: 12345}
     );
 
-    expect(config.settings).toMatchObject({
+    expect(resolvedConfig.settings).toMatchObject({
       formFactor: 'mobile', // inherit from default
-      output: 'csv', // config-specific overrides
+      output: 'csv', // resolvedConfig-specific overrides
```
Comment (connorjclark) [inline core/test/config/config-test.js:62]: intentional ?

File: core/test/config/config-test.js [hunk 2]
```diff
@@ -104,33 +104,33 @@ describe('Fraggle Rock Config', () => {
...
     expect(auditIds).toContain('document-title'); // from onlyCategories
     expect(auditIds).not.toContain('first-contentful-paint'); // from onlyCategories
     expect(auditIds).not.toContain('robots-txt'); // from skipAudits
   });
 
   it('should support plugins', async () => {
-    const {config} = await initializeConfig('navigation', undefined, {
-      configPath: `${LH_ROOT}/core/test/fixtures/config-plugins/`,
+    const {resolvedConfig} = await initializeConfig('navigation', undefined, {
+      configPath: `${LH_ROOT}/core/test/fixtures/resolvedConfig-plugins/`,
```
Comment (connorjclark) [inline core/test/config/config-test.js:129]: intentional?

**Review by connorjclark — APPROVED** [review]

File: core/test/config/config-test.js (hunk 1 above)
Comment (adamraine) [reply core/test/config/config-test.js:62]: No

File: core/test/config/config-test.js (hunk 2 above)
Comment (adamraine) [reply core/test/config/config-test.js:129]: No

**Pull request merged**